frontend/Container.js

```diff
@@ -19,6 +19,15 @@ var TabbedPane = require('./TabbedPane');
 
 import type MenuItem from './ContextMenu';
 
+type Props = {};
+
+type State = {
+  isVertical: boolean,
+};
+
+var IS_VERTICAL_BREAKPOINT = 500;
+var resizeTimeout = null;
```

Comment:
We should still store it on the instance.
If Flow complains, you need to explcitly declare it:

```
  resizeTimeout?: number;
```

Just like we declare `props` and `state` types inside the class.